moveit_core/collision_detection/include/moveit/collision_detection/world.h

```diff
@@ -175,6 +203,14 @@ class World
    * Returns true on success and false if no such object was found. */
   bool removeObject(const std::string& object_id);
 
+  /** \brief Set subframes on an object. */
+  bool setSubframesOfObject(const std::string& object_id,
+                              const std::map<std::string, Eigen::Isometry3d>& subframe_poses);
```

Comment:
A map with non-aligned Eigen objects should not be used at all. I suggest to keep the second version only.

Reply:
Alright. Will remove tomorrow.

Reply:
Done in my fork.